R/pacta.portfolio.report-package.R

```diff
@@ -0,0 +1,7 @@
+#' @keywords internal
+"_PACKAGE"
+
+## usethis namespace: start
+#' @importFrom dplyr reframe
+## usethis namespace: end
+NULL
```

Comment:
sorry retrospectively, but... seems off that this file is necessary for just `dplyr::reframe`, probably suggests that this is unnecessarily a new and unique method (to this repo) of importing a function

Reply:
My preferred method is usually to use an `imports.R` file: https://github.com/RMI-PACTA/r2dii.analysis/blob/main/R/imports.R

but that doesn't seem to be the Beta in `pacta.portfolio.report`? Unless there is a different approach already implemented here?

Reply:
It looks like we were already using the R package standard of putting `importFrom`s in NAMESPACE, but `usethis::use_import_from()` requires having the package-level docs.